tests/functional/README.md

```diff
@@ -1,18 +1,38 @@
 Running the Functional Tests
-==
+============================
+
+Functional tests run against a set of Docker containers that mimic the AWS stack this service runs on
+in production.
+
+To run the functional tests with a Python 2.7 image, do this:
 
 ```bash
-> docker build -t fsm -f tests/functional/Dockerfile .
-> docker build -t fsm_docker_runner -f tools/experimental/Dockerfile.fsm_docker_runner .
-> docker build -t dev_ecs -f tools/experimental/Dockerfile.dev_ecs .
-> docker run \
+$ docker build -t fsm_docker_runner -f tools/experimental/Dockerfile.fsm_docker_runner .
+$ docker build -t dev_ecs -f tools/experimental/Dockerfile.dev_ecs .
+$ docker build -t fsm -f tests/functional/Dockerfile .
+$ docker run \
   -p 11211:11211 -p 6379:6379 -p 4568:4568 \
-  -v /var/run/docker.sock:/var/run/docker.sock fsm \
+  -v /var/run/docker.sock:/var/run/docker.sock \
   -e VOLUME=${PWD}/tests/functional/settings.py.functional:/usr/local/bin/settings.py \
   -e LINK=fsm \
   --hostname fsm \
-  --name fsm \
```

Comment:
This caused an error when you run this command a second time, as it conflicts with the container created on first run. I found that the tests ran fine without this.

Reply:
the value of `-e LINK=fsm` needs to match this. somehow we should adjust this command to generate a token or something. the `make function` works, but there is an undocumented functional suite that just run in the container, and it (so far) only verifiable by watching the tests run.

Reply:
with this change, you see an error `docker: Error response from daemon: Cannot link to a non running container: /fsm AS /nostalgic_goldberg/fsm.`

Reply:
or maybe just delete the previous container (by name) every time...

Reply:
We'll have to work out a better solution for this, "undocumented functional suite" with manual verification is not great. Neither is an error on second run.

Reply:
please add `docker container rm fsm` before the `run`. add a comment that that must be run every time.